### PR TITLE
PCX-1635: content types for terraform and randomness beacon

### PR DESCRIPTION
**Pull request by nayaabahsan**
PCX content type labelling for terraform and randomness beacon

### CODE REVIEW

**Review by lukevalenta — APPROVED** [review]

**Review by armfazh — APPROVED** [review]

File: products/randomness-beacon/src/content/cryptographic-background/randomness-generation.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 order: 1
+pcx-content-type: tutorial
```
Comment (crwaters16) [inline products/randomness-beacon/src/content/cryptographic-background/randomness-generation.md]: We may want to label this as "interim" for now - it doesn't follow the flow I'd expect of a tutorial.

File: products/randomness-beacon/src/content/cryptographic-background/setup-phase.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 order: 0
+pcx-content-type: concept
```
Comment (crwaters16) [inline products/randomness-beacon/src/content/cryptographic-background/setup-phase.md]: Similar comment here - this feels like a combination of how-to, reference, and concept info.

File: products/randomness-beacon/src/content/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Welcome
```
Comment (crwaters16) [inline products/randomness-beacon/src/content/index.md]: Update to Overview

File: products/randomness-beacon/src/content/index.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
 order: 0
-type: overview
+pcx-content-type: landing-page
 ---
 
 # Cloudflare Randomness Beacon docs
```
Comment (crwaters16) [inline products/randomness-beacon/src/content/index.md]: Delete "docs" from H1 title

File: products/terraform/src/content/index.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Welcome
```
Comment (crwaters16) [inline products/terraform/src/content/index.md]: Update to Overview (and remove "documentation" from H1 title)

**Review by crwaters16 — APPROVED** [review]
Left a few comments - part of me wants to mark all those Terraform tutorials as "interim" because they vary in how they're structured, but given they're in a tutorial subsection, leaving as-is is fine for now.

**Pull request merged**